dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/SecurityOwnershipValidator.java

```diff
@@ -113,13 +114,13 @@ public void validate(
     // has to be checked
     if (program.isWithoutRegistration() || strategy.isCreate() || strategy.isDelete()) {
       if (organisationUnit == null) {
-        log.warn("Event " + event.getEvent() + ORG_UNIT_NO_USER_ASSIGNED);
+        log.warn(String.format(ORG_UNIT_NO_USER_ASSIGNED, event.getUid()));
```

Comment:
Use parameterized logging https://www.slf4j.org/faq.html#logging_performance. We should use that in general to avoid building any structures that are not needed. If log level is set to error the string format computation would be wasted.